Curve-finance-suite/src/agents/curve.dao.claim.many.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import Web3 from 'web3';
+import abi from '../utils/fee.distribution';
+import {
+  provideFunctionCallsDetectorAgent,
+  FindingGenerator,
+} from 'nethermindeth-general-agents-module';
+
+// @ts-ignore
+import abiDecoder from 'abi-decoder';
+abiDecoder.addABI(abi);
+
+export const web3 = new Web3();
+
+export const claimMany = {
+  name: 'claim_many',
+  outputs: [{ type: 'bool', name: '' }],
+  inputs: [{ type: 'address[20]', name: '_receivers' }],
+  stateMutability: 'nonpayable',
+  type: 'function',
+  gas: 26281905,
+};
+
+const createFinding = (alertId: string): Finding =>
+  Finding.fromObject({
+    name: 'Claim Rewards function called',
+    description: 'Claim Rewards function called on pool',
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?